src/components/Apps/index.jsx

```diff
@@ -50,9 +49,7 @@ class Apps extends Component {
     const { opened } = this.state
     return (
       <nav
-        className={`coz-nav coz-nav-apps${
-          replaceTitleOnMobile ? ' coz-bar-hide-sm' : ''
-        }`}
+        className='coz-nav coz-nav-apps'
```

Comment:
Is it a breaking change? It seems that in Drive / Photo we use this `replaceTitleOnMobile` props 🤔

Reply:
There was no difference in behavior if I removed or kept the prop.